.gitignore

```diff
@@ -3,3 +3,8 @@ public
 ._.DS_Store
 ._*
 node_modules
+scratch/
+.ropeproject
+*.pyc
+ve/
+*.swp
```

Comment:
btw - for some file patterns that you never want to commit, like `*.swp`, it might save you some time to make a global gitignore and add this there: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Yeah, good idea, will do.